pygmt/tests/test_clib_loading.py

```diff
@@ -208,6 +209,45 @@ def test_brokenlib_brokenlib_workinglib(self):
         assert check_libgmt(load_libgmt(lib_fullnames=lib_fullnames)) is None
 
 
+class TestLibgmtCount:
```

Comment:
Maybe better to put this unit test in `test_session_management.py`?

Reply:
Prefer to keep it in `test_clib_loading.py` because this unit test is actually not related to session management.

Reply:
Ok, I kinda wanted to move the test since `test_clib_loading.py` has 300+ lines of code, while `test_session_management.py` has <100 lines, and we're kinda checking that `Session()` doesn't reload libgmt here, but up to you :slightly_smiling_face: